src/lib/libast/misc/getenv.c

```diff
@@ -21,13 +21,13 @@
 ***********************************************************************/
 #pragma prototyped
 
-#if _UWIN && __STDPP__
+#if __STDPP__
```

Comment:
You introduced a bug. The original was of the form `a && b`. Removing `a` is equivalent to setting it to false and thus the entire expression is false. I'll take care of fixing this when I merge it.

Reply:
And also similarly on line 30.